tests/base/function_parser_05.cc

```diff
@@ -85,7 +85,9 @@ test2()
 
   // Output the evaluated function
   deallog << "Function '" << expressions[0] << "," << expressions[1] << "'"
-          << " @ " << point << " is " << result << std::endl;
+          << " @ " << point << " is " << std::flush;
+  result.print(deallog.get_file_stream(), 4, false);
```

Comment:
Is this a mistake? I thought you were *removing* the `print` function.

Reply:
We are removing the overload taking `LogStream` but we still allow `std::ostream`.

Reply:
Or are you saying that we shouldn't remove the `operator<<(LogStream,...)` function that is implemented in terms of `print` as well?

Reply:
No, I was just confused why we are moving from `operator<<` to `results.print(...)`. I hadn't realized that you were also removing `operator<<(LogStream)`.

But if it were up to me, we'd completely remove `LogStream` to begin with...